lambda_handlers/register_module_handler.py

```diff
@@ -20,4 +20,5 @@ def _handle(self, event, context):
         vars = {}
```

Comment:
Shouldn't use vars as a variable name because it is redefining a built-in Python function.